cloud-controller-manager/osc/osc_test.go

```diff
@@ -856,11 +854,11 @@ func TestSubnetIDsinVPC(t *testing.T) {
 	for _, rt := range constructedRouteTables {
 		awsServices.compute.CreateRouteTable(rt)
 	}
-	request1111 := &ec2.DescribeSubnetsInput{}
+	request1111 := &osc.ReadSubnetsRequest{}
 	res, _ := awsServices.compute.DescribeSubnets(request1111)
 	t.Logf("awsServices.ec2.DescribeSubnets----: %v", res)
 
-	request2222 := &ec2.DescribeRouteTablesInput{}
+	request2222 := &osc.ReadRouteTablesRequest{}
```

Comment:
request2222 fun name :)

Reply:
kept the name used, not conventional though